explorer/explorer.go

```diff
@@ -402,6 +402,11 @@ func (e *Explorer) Hosts(pks []types.PublicKey) ([]Host, error) {
 	return e.s.QueryHosts(HostQuery{PublicKeys: pks}, HostSortPublicKey, HostSortAsc, 0, math.MaxInt64)
 }
 
+// HostNetAddresses returns the hosts with the specified net addressess.
+func (e *Explorer) HostNetAddresses(netAddresses []string) ([]Host, error) {
+	return e.s.QueryHosts(HostQuery{NetAddresses: netAddresses}, HostSortPublicKey, HostSortAsc, 0, math.MaxInt64)
```

Comment:
I think this method could have a little more logic to deal with no hosts being found, or potentially handle the case where more than one host is found. E.g. we could limit by one and perform the `len(hosts)` check here.